src/Features/Traits/Dispatcher.php

```diff
@@ -4,6 +4,97 @@
 
 namespace Blumilk\BLT\Features\Traits;
 
+use Behat\Gherkin\Node\TableNode;
+use Blumilk\BLT\Helpers\RecognizeClassHelper;
+use Blumilk\BLT\Helpers\TypesEnum;
+use Illuminate\Contracts\Bus\Dispatcher as BusDispatcher;
+use Illuminate\Contracts\Container\BindingResolutionException;
+use Illuminate\Contracts\Events\Dispatcher as EventDispatcher;
+use Illuminate\Support\Testing\Fakes\BusFake;
+use Illuminate\Support\Testing\Fakes\EventFake;
+use InvalidArgumentException;
+
 trait Dispatcher
 {
+    use Application;
+
+    private BusFake $busFake;
```

Comment:
Please reconsider changing these to protected. If anyone would like to extend our code it would be easier for them.